src/foam/nanos/session/services

```diff
@@ -36,7 +36,7 @@ p({
 p({
   "class": "foam.nanos.boot.NSpec",
   "name": "sessionService",
-  "serve": true,
+  "serve": false,
```

Comment:
The app works with this?  The whole point of the sessionService is for the client.  This suggests we don't need this service.